web/src/client/manager.js

```diff
@@ -123,6 +123,20 @@ class ManagerBaseClient {
   rebootSystem() {
     return cockpit.spawn(["/usr/sbin/shutdown", "-r", "now"]);
   }
+
+  /**
+   * Returns whether finishing cockpit succeeded
+   */
+  finishCockpit() {
+    return cockpit.spawn(["systemctl", "stop", "cockpit-wsinstance-http.service"]);
```

Comment:
Does this really work? Systemd is probably included as container is based on tumbleweed one, but it's not explicitly included. What happens when base container switches to BCI one or ALP based one? Is systemd required in agama containers?

Also cockpit-ws is started directly, not as a service. At least that's how I see it in [here](https://build.opensuse.org/package/view_file/YaST:Head:Containers/agama-web/run_background_agama.sh?expand=1)

Reply:
Good point. Actually, neither my proposal to move that logic to the service would work (because it runs on a different container).

Reply:
discussed off the github. We went with moving to backend, changing command for restarting and some more changes on iguana side (workflow file update)